README.md

```diff
@@ -48,41 +48,67 @@ Questionable Conditions that the application will verify with User via A Dialog
 - Output File does not end with .pdf, verify with user that's really what they want
 - Output File Name Matches a File in the Output Directory
 
+
+---
+
+## Beofre Installation
+
+### 1. Download Python
+Users are required to have Python3 installed in their machine before using the application.
+(Please provide a specific way of installing, either through Terminal/should they consider using bash?)
+
+
+### 2. Grant access
+(Reason behind this move shall be a reasonable move to ease new users without any ideas what they are getting into)
+![Allow](https://osi.3df.io/wp-content/uploads/2018/05/MacSecuritySettings.png)
+
+If the option *Anywhere* is missing, open Terminal.app and type in the following command.
+![Missing Anywhere Option](https://github.com/ivanoung/iDensify/blob/master/img/nodeveloperoption.png?raw=true)
```

Comment:
There's an easier way to do that which is to click on the lock icon on the bottom left and all the missing config will show up. The bottom left icon will also turn into an unlocked pad lock. Interesting how you derived a techie alternative instead (ie. CLI solution)

Reply:
The lock was the first thing I tried. I was working with a freshly formatted Mac from my neighbor, and all I was doing was following the basic visual cues that Apple provided me to find out the answer to my question. The lock was my initial attempt; I only went to Google after it failed to reveal the extra option.

Reply:
Regardless, I think it’s good that this is there then if there’s a chance that unlocking doesn’t work. Good alternative. Perhaps it should be in its own troubleshooting section?

Reply:
That could work as well, without disturbing the actual instruction and diverting users from their initial purpose.